src/messaging.js

```diff
@@ -209,6 +209,10 @@ var async = require('async'),
 			};
 
 			plugins.fireHook('filter:messaging.parse', messageData, function(err, messageData) {
+				if (err) {
```

Comment:
remove this, I will have to fix this function first before adding err, looks like the function is sending the result in the first param as well.